### DISCUSSION TITLE
how to use RFM95 with Blue pill SPI2

### DISCUSSION THREAD

**Discussion by fizga** [jgromes/RadioLib#1058] [General]
hello,
I am a beginner with STM32s and electronics is my hobby.
someone could tell me how to use an RFM95 on the SPI2 of a blue pill, I have already used the RFM95 with arduinos but never with blue pill, an example of code would be ideal to understand.
while doing research on the net I saw some posts on how to declare specific pins but it is not clear to me.

**Comment by HeadBoffin** [comment]
Searching the discussions & issues has some examples. Plus the wiki page: https://github.com/jgromes/RadioLib/wiki/Basics#non-standard-spi-setup
